### PR TITLE
Add Bootstrap

### LINKED ISSUE #12: Setup SASS + Bootstrap

### PR DESCRIPTION
**Pull request by martomi**
Close #12

### CODE REVIEW

**Review by lustoykov — DISMISSED** [review]
My preference is to use latest stable version (no alpha). We can't afford to waste time, if eventually something is buggy.

**Comment by chaoran-chen** [comment]
According to @kordianbruck, it seems to be quite stable, and it has with Flexbox a nice new feature. I think that we should give it a try.

But another point: do you think that we should add *.css and *.css.map to the repository since we will be coding in sass?

**Comment by kordianbruck** [comment]
Don't add any generated files. Sass usually creates those when you run the build. Otherwise you will be updating a lot of code with each commit, which is not nessecary

**Comment by lustoykov** [comment]
Ok, let's give it a try then

File: .gitignore [hunk 1]
```diff
@@ -2,6 +2,11 @@
 logs
 *.log
 npm-debug.log*
+warnings.json
+
+# Generated styles
+style.css
+style.css.map
```
Comment (lustoykov) [inline .gitignore]: `*.css, *.css.map` doesn't work?

Comment (chaoran-chen) [reply .gitignore]: I think that it is good so. Maybe we will later add external CSS libraries which are not on bower/npm.

**Review by chaoran-chen — APPROVED** [review]
From my side, it is okay.

File: .gitignore (hunk 1 above)
Comment (martomi) [reply .gitignore]: Yes, let's not apply masks before we actually need it.

Comment (lustoykov) [reply .gitignore]: Sorry my bad! Totally right!

**Comment by sacdallago** [comment]
It's the first time in three runs that this PR story goes as I imagined it from day 0 🎊

**Comment by kordianbruck** [comment]
@sacdallago this time we got amazing tutors - right? :peace_symbol:

**Pull request merged**